DigitalLearningSolutions.Web/Controllers/LearningContentController.cs

```diff
@@ -11,36 +12,24 @@
     [SetSelectedTab(nameof(NavMenuTab.Welcome))]
     public class LearningContentController : Controller
     {
-        private const string ItSkillsPathwayBrand = "ITSkillsPathway";
-        private const string ItSkillsPathwayTitle = "IT Skills Pathway";
-        private const string ReasonableAdjustmentFlagBrand = "ReasonableAdjustmentFlag";
-        private const string ReasonableAdjustmentFlagTitle = "Reasonable Adjustment Flag";
+        private readonly IBrandsService brandsService;
 
-        private const string TerminologyAndClassificationsDeliveryServiceBrand =
-            "TerminologyandClassificationsDeliveryService";
-
-        private const string TerminologyAndClassificationsDeliveryServiceTitle =
-            "Terminology and Classifications Delivery Service";
-
-        public IActionResult ItSkillsPathway()
+        public LearningContentController(IBrandsService brandsService)
         {
-            var model = new LearningContentViewModel(ItSkillsPathwayBrand, ItSkillsPathwayTitle);
-            return View("Index", model);
+            this.brandsService = brandsService;
         }
 
-        public IActionResult ReasonableAdjustmentFlag()
+        [Route("LearningContent/{brandId:int}")]
+        public IActionResult BrandPageDeliveryService(int brandId)
         {
-            var model = new LearningContentViewModel(ReasonableAdjustmentFlagBrand, ReasonableAdjustmentFlagTitle);
-            return View("Index", model);
-        }
+            var brand = brandsService.GetPublicBrandById(brandId);
+            if (brand == null)
+            {
+                return NotFound();
+            }
+
+            var model = new LearningContentViewModel(brand);
 
-        public IActionResult TerminologyAndClassificationsDeliveryService()
-        {
-            var model = new LearningContentViewModel(
-                TerminologyAndClassificationsDeliveryServiceBrand,
-                TerminologyAndClassificationsDeliveryServiceTitle,
-                true
-            );
             return View("Index", model);

```

Comment:
If you change this controller action name to "Index", you should be able to remove the View name "Index" from this line :)